compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5441,8 +5441,12 @@ static TR::Register *inlineArrayCmpP10(TR::Node *node, TR::CodeGenerator *cg)
 }
 
 
-static TR::Register *inlineArrayCmp(TR::Node *node, TR::CodeGenerator *cg)
+static TR::Register *inlineArrayCmp(TR::Node *node, TR::CodeGenerator *cg, bool isArrayCmpLen)
    {
+   static char *disableP10ArrayCmp = feGetEnv("TR_DisableP10ArrayCmp");
+   if (cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P10) && (disableP10ArrayCmp != NULL))
```

Comment:
I see, this is fixed in the second commit. Would you mind correcting the first commit instead? Better to avoid introducing a bug than to introduce it and then fix it